deliverables/README.md

```diff
@@ -1,68 +1,140 @@
-What is the role of HTML in a web page?  
-What is the role of CSS in a web page?  
-What is the role of Javascript in a web page?  
+#### Required - Answer these questions in the deliverables file, and complete the following.
+
+What is the role of HTML in a web page?
+HTML is the language, or code, used to edit and position the text, images, frames and other web page elements. Its purely for content.
+What is the role of CSS in a web page?
+CSS is an enhanced form of HTML, and makes it easier to edit the design of a web page.
+What is the role of Javascript in a web page?
+Java script primarily to make a website more active or visual, it can not go without html / css and works with it but is seperate from html and css.
 
 ---
 
 ### Section 2: HTML and CSS 101
 
 #### meta
-Why do we separate HTML and CSS?  
+Why do we separate HTML and CSS?
 
+That way it is easy to distinguish the content we are editing rather than the design of the code.
 #### HTML
-What are classes and IDs (and how are they different)?  
-What are elements?  
-What are tags?  
-What are attributes?  
-What are forms?  
-What is a div?  
-
-What are selectors?  
-What are properties?  
-What are values?  
-How do CSS styles for a particular element get inherited? ie. how does an element get its "default" styles?  
-What are two CSS attributes you can change to push an element around on the page?  
-What are the three different ways to include CSS in your project or use CSS to style a particular element?  
+What are classes and IDs (and how are they different)?
+ID's are unique throughout the code and allow you to label and edit one aspect while Classes are not unique and allow you to access and edit more than one portion of code at a time.
+
+What are elements?
+Elements are used to organize the components of code in a web page. They always have two tags, a beginning and end.
+
+What are tags?
+Tags are used to define the HTML and apply something to the content.
+
+What are attributes?
+There are many different kind of attributes that can go inside a tag to modify the current tag.
+
+What are forms?
+Forms are used within webpages to make the website more interactive and gather information, it takes some kind of information and sends it to some kind of database.
+
+What is a div?
+Div tag works as a container for HTML code and can be used to combine / label code.
+
+What are selectors?
+In CSS, selectors are patterns used to select the element(s) you want to style.
+
+What are properties?
+Code can have properties just like a dog has fur, a tail, and eye color.
+
+What are values?
+The variable name is the usual way to reference the stored value; this separation of name and content allows the name to be used independently of the exact information it represents. The identifier in computer source code can be bound to a value during run time, and the value of the variable may thus change during the course of program execution
```

Comment:
I see this is copy-pasted from http://en.wikipedia.org/wiki/Variable_%28computer_science%29  

It's fine to copy-paste content, as long as you follow a few rules:  
* make sure it actually answer the question.  In this case it doesn't, because here we're talking specifically about CSS values, not the more general concept.  
* include a link to the source